DOC Update project history and infrastructure support sections (#1342)
iodide-project/pyodide: docs/project/about.md

the Python scientific stack including NumPy, Pandas, Matplotlib, parts of
SciPy, and NetworkX. The [packages
directory](https://github.com/iodide-project/pyodide/tree/master/packages)
lists over 75 packages which are currently available. In addition it's possible
to install pure Python wheels from PyPi.

Pyodide provides transparent conversion of objects between Javascript and
Python. When used inside a browser, Python has full access to the Web APIs.

## History

> Pyodide was created in 2018 by [Michael Droettbom](https://github.com/mdboom)

Reviewer: ```suggestion
Pyodide was created in 2018 by [Michael Droettboom](https://github.com/mdboom)
```

> at Mozilla as part of the [iodide project](https://iodide.io), a set of experiments around
> scientific computing and communication for the web.

Reviewer 2: It might be better to link to the GitHub repository. I don't know how much longer iodide.io will be up but I suspect pyodide and the relevance of this document will outlive it. 😄 

```suggestion
at Mozilla as part of the [iodide project](https://github.com/iodide-project/iodide), a set of experiments around
scientific computing and communication for the web.
```

Author: Thanks @wlach !

At present Pyodide is an independent and community driven open-source project.
The decision making process is outlined in the {ref}`project governance <project-governance>`.
Pyodide may be used standalone in any context where you want to run Python
inside a web browser.

## Infrastructure support

We would also like to thank,
 - [Mozilla](https://www.mozilla.org/en-US/) and

> [CircleCl](https://circleci.com/) for Continuous Integration resources

Reviewer: ```suggestion
- [CircleCl](https://circleci.com/) for Continuous Integration resources
```

Author: That was meant to be a single bullet point, but we can certainly split it if there are more thing to put for Mozilla support :)

More seriously, we can split if you have suggestions what to put there because I think,
```
 - Mozilla and
 - CircleCI for Continuous Integration resources
```
looks a bit strange.

Reviewer: oh sorry, i was quickly scanning over it and didn't see the and at the end of the line. you can discard this :)
